### DISCUSSION TITLE
Unable to see 120p res option in 100ms dashboard

### DISCUSSION THREAD

**Discussion by robillo** [100mslive/sample-app-android#98] [Q&A]
Have heard products using 100ms 120p res video streaming but unable to see any res option below 240p on 100ms dashboard. Any way to configure for 120p res?

**Comment by PratimMallick** [comment]
@robillo It is in internal QA testing right now. It will be available as soon as the release happens (latest by 15th sept).

The products running on 120p are using our v1, which do not have the settings in dashboard

**Reply by robillo** [reply]
That's great. Thanks for the update.
